<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Resultado</class>
 <widget class="QWidget" name="Resultado">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>466</width>
    <height>760</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>466</width>
    <height>760</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>466</width>
    <height>760</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="windowFilePath">
   <string notr="true"/>
  </property>
  <widget class="QLabel" name="contadorLabel">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>430</y>
     <width>311</width>
     <height>101</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QCalendarWidget" name="calendarWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>90</y>
     <width>441</width>
     <height>331</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>999</width>
     <height>999</height>
    </size>
   </property>
  </widget>
  <widget class="QLabel" name="respuestasLabel">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>580</y>
     <width>311</width>
     <height>101</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
